integration-tests/common/common.go

```diff
@@ -148,8 +148,9 @@ func (c *Common) CreateJobsForContract(cc *ChainlinkClient, observationSource st
 
 	// Defining relay config
 	relayConfig := job.JSONConfig{
-		"nodeName": fmt.Sprintf("\"starknet-OCRv2-%s-%s\"", "node", uuid.NewV4().String()),
-		"chainID":  fmt.Sprintf("\"%s\"", c.ChainId),
+		"nodeName":       fmt.Sprintf("\"starknet-OCRv2-%s-%s\"", "node", uuid.NewV4().String()),
+		"accountAddress": fmt.Sprintf("\"%s\"", c.Account),
```

Comment:
so i'm not confident `c.Account` is what you are looking for? (i'm not sure what that references)

having it in this struct means that it would be the same for each node - but you would want it to be different for each one...I think it would probably be somewhere in the for-loop that is further down in the file?